src/Engine/Renderer/Texture/Texture.hpp

```diff
@@ -186,9 +188,9 @@ class RA_ENGINE_API Texture final {
      * This will transform the internal representation of the texture to GL_FLOAT.
      * Only GL_RGB[8, 16, 16F, 32F] and GL_RGBA[8, 16, 16F, 32F] are managed.
      * Full transformation as described at https://en.wikipedia.org/wiki/SRGB
-     * @param gamma the gama value to use (sRGB is 2.4)
+     * The gama value  used is the sRGB one  2.4
```

Comment:
2.2 ;)

Reply:
from sRGB wikipedia page : 
> Unlike most other RGB color spaces, the sRGB gamma cannot be expressed as a single numerical value. The overall gamma is approximately 2.2, consisting of a linear (gamma 1.0) section near black, and a non-linear section elsewhere involving a 2.4 exponent and a gamma (slope of log output versus log input) changing from 1.0 through about 2.3.

So, the gamma value 2.4 is for the non linear part, the gamma value 1 is the value of the lienear part

Concl. thereis no parameter and this comment will be updated to remove all numeric precision and to give only the link.